resources/assets/js/app/views/topics/topicLayout.js

```diff
@@ -1,4 +1,15 @@
 var Marionette = require('backbone.marionette');
 var logger = require('../../instances/logger');
+var topicCollection = require('./topicCollection');
 
-module.exports = Marionette.LayoutView.extend({});
\ No newline at end of file
+module.exports = Marionette.LayoutView.extend({
+    template: 'topicLayout',
+    regions: {
+        container: '#posts'
+    },
+    onShow: function () {
```

Comment:
use onRender, if this view will be rerendered because of some event, collection won't be shown